rails/benchmarks/activemodel_views.rb

```diff
@@ -37,7 +39,7 @@ def user_factory
   { email: Faker::Internet.email }
 end
 
-def render_views
+m = Benchmark.rails(100, "actionview/render_activemodels") do
```

Comment:
I used kwargs for other arguments and I think it's not necessary for first 2 required args.